packages/cli-e2e/__tests__/auth.specs.ts

```diff
@@ -24,9 +24,14 @@ describe('auth', () => {
 
     it('should open the platform page', async () => {
       // TODO CDX-98: Remove `-e=dev`.
+      const args: string[] = [CLI_EXEC_PATH, 'auth:login', '-e=dev'];
+      if (process.platform === 'win32') {
```

Comment:
Are those platform string typed in any manner ? 

If not, a constant from node that we could use ?

Reply:
the constants are typed, but not only typed (i.e. it's a `declare const` but it's not actually exposed afaik)